projectq/meta/_control.py

```diff
@@ -24,18 +24,24 @@
 """
 
 from projectq.cengines import BasicEngine
-from projectq.meta import ComputeTag, UncomputeTag
+from projectq.meta import ComputeTag, UncomputeTag, Compute, Uncompute
 from projectq.ops import ClassicalInstructionGate
 from projectq.types import BasicQubit
+from projectq.ops import _command, X
 from ._util import insert_engine, drop_engine_after
+from enum import Enum
+
+class State(Enum):
+    AllZero = 0
+    AllOne = -1
```

Comment:
```suggestion
    AllOne = 1
```

Since we have an enum, the actual value does not matter, so I would rather have `1` for `AllOne`